docroot/.htaccess

```diff
@@ -69,6 +69,20 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
+  # Forbid access to install and update files.
+  RewriteRule ^(core/install|update)\.php - [F]
+
+  # Forbid access to admin, node, and user paths on domains other than admin.foia.gov and local.dojfoia.gov.
```

Comment:
Comment needs updating since we're not specifying our local domain any longer. Maybe we can keep it generic in terms of the paths as well, in case we add more down the road.